static/src/javascripts/projects/common/modules/experiments/tests/across-the-country.js

```diff
@@ -0,0 +1,53 @@
+define([
+    'fastdom',
+    'common/utils/$',
+    'common/utils/config'
+], function (
+    fastdom,
+    $,
+    config
+) {
+
+    return function () {
+        this.id = 'AcrossTheCountry';
+        this.start = '2015-03-19';
+        this.expiry = '2015-04-19';
+        this.author = 'Grant Klopper';
+        this.description = 'Checking effect of moving "across the country" container higher up the US front';
+        this.audience = 1.0;
+        this.audienceOffset = 0;
+        this.successMeasure = '';
+        this.audienceCriteria = 'All users';
+        this.dataLinkNames = '';
+        this.idealOutcome = 'More people read content off of the US front';
+
+        this.canRun = function () {
+            return config.page.pageId === 'us';
+        };
+
+        /**
+         * nothing happens in here, we just use this to bucket users
+         */
+        this.variants = [
+            {
+                id: 'control',
+                test: function () { }
+            },
+            {
+                // this is all below the fold.
+                id: 'variant',
+                test: function () {
+                    var acrossTheCountryContainer = $('#across-the-country'),
+                        highlightsContainer = $('#highlights');
+                    if (acrossTheCountryContainer.length === 1 && highlightsContainer.length === 1) {
+                        fastdom.write(function () {
+                            acrossTheCountryContainer.remove();
+                            acrossTheCountryContainer.insertAfter(highlightsContainer);
```

Comment:
Success is not "do more people click on across the country container" that would obviously happen by moving it up the page.

Success is "does the front page as a whole have a higher click through rate".